### DISCUSSION TITLE
image with reference docker.io/wardenenv/mysql:8.0 was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64

### DISCUSSION THREAD

**Discussion by janssensjelle** [wardenenv/warden#656] [Help & Support] [answered]
So today I am experiencing the following issue:

image with reference docker.io/wardenenv/mysql:8.0 was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64

These images exist (also occurs for PHP, elasticsearch etc), can't seem to find a reason why it does this today. Docker and Warden are both the latest version.

I have got this in all my projects - with different versions of everything.

**Comment by navarr** [comment, marked as answer]
Double check that you only have one installation of Warden and that it is up to date.  `which warden` should show where the Warden that you are running is installed

**Reply by janssensjelle** [reply]
There's only one instance of Warden, which was installed via Homebrew. At this moment the issues doesn't seem to occur anymore.
